apps/frontend/src/libs/components/header/header.tsx

```diff
@@ -0,0 +1,30 @@
+import { getAvatarInitials } from "~/libs/helpers/helpers.js";
+
+import styles from "./styles.module.css";
+
+const Header = (): JSX.Element => {
+	const name = "Daniel"; // This is developer data to test getAvatarInitials function
```

Comment:
take the user from the state instead

Reply:
I've taken the list of users from the store and used the first entry. I'll wait for the getAuthenticatedUser function to be created to be able to directly obtain the user.